grtext/grtext.cpp

```diff
@@ -230,10 +230,13 @@ void grtext_Init() {
 }
 
 //	macro to get character width
+#undef CHAR_WIDTH
```

Comment:
where would that definition come from? If it is from a system include, it could be a good idea to change the name of our functions to something else instead

Reply:
See [Numeric limits](https://en.cppreference.com/w/c/types/limits) since C23
Yes, probably using #ifndef should be sufficient.

Reply:
Probably renaming to GRCHAR_WIDTH should do the job